src/main/aspect/com/arpnetworking/steno/aspect/LogBuilderAspect.java

```diff
@@ -39,7 +39,7 @@
     public void addToContextLineAndMethod(final JoinPoint joinPoint) {
         final SourceLocation sourceLocation = joinPoint.getSourceLocation();
         final LogBuilder targetLogBuilder = (LogBuilder) joinPoint.getTarget();
-        targetLogBuilder.addContext("line", sourceLocation.getLine());
+        targetLogBuilder.addContext("line", String.valueOf(sourceLocation.getLine()));
```

Comment:
Actually it always should have been a string. In this change I added schema validation to the "integration" tests and this bug fell out. In all the v0 Steno schema references I have line number is a string. Obviously, not entirely clear what the authors had in mind but at least for the time being I believe the library should stick to the spec. If v1 is ever released then we can consider forking at that time (I fear they may take the format in a different direction than what we started with).